source/includes/stackpath/_workloads.md

```diff
@@ -363,20 +452,26 @@ curl -X PUT \
   "name": "my-container-workload",
   "type": "CONTAINER",
   "image": "redis:alpine",
-  "environmentVariableKey": "env-key",
-  "environmentVariableValue": "env-value",
-  "secretEnvironmentVariableKey": "secret-key",
-  "secretEnvironmentVariableValue": "secret-value",
+  "secretEnvironmentVariables": [
+    {
+      "key": "PASSWORD",
+      "value": "my-password"
```

Comment:
We should put one with [REDACTED] to explain it will not remove and will not use that value.

Reply:
Added an inline comment and noted it in the attributes section as well.